File: Mouse.pl
#Mouse

use 5.022;

my $xy = <>;
my $click = <>;
my $sign = <>;

chomp $xy;
chomp $click;
chomp $sign;

my ($x,$y) = map {int} split /,/, $xy;
$click = 1 if $click eq 'true';
$click = 0 if $click eq 'false';

say $x . ',' . $y;
say $click;
say $sign;


# Tovarisch Trunaev, GPU Officer Yahwe, Russia, Uray, 2026
# $Om->Delf();

File: Mouse.pas
Program Mouse;
uses SysUtils;
Var
  xy, click, sign : string;
  x, y, c, i: integer;
Begin
  ReadLn(xy);
  ReadLn(click);
  ReadLn(sign);
  
  i := Pos(xy, ',');
  x := StrToInt(Copy(xy, 1, i - 1));
  y := Round(StrToFloat(Copy(xy, i + 1, Length(xy))));
  
  If click = 'true' Then
	c := 1;
  Else If click = 'false' Then
    c:= 0;
  Else
    c := StrToInt(click);
  
  WriteLn(x, ',', y);
  WriteLn(c);
  WriteLn(sign);
End.

{ Tovarisch Trunaev, GPU Officer Yahwe, Russian, Uray }
{ Om->Delf() }
